lastfm/js/test.js

```diff
@@ -0,0 +1,60 @@
+var assert = chai.assert;
+
+describe("getRequest.doGet", function () {
+  it("(getRequest.doGet(api.searchArtist('Cher'), function(){}) exec onload", function (done) {
+    http.doGet(api.searchArtist('Cher'), function () {
```

Comment:
actually sinon is required to spy on methods
common practise for mocha: mocha + chai + sinon